tests/components/dlna_dmr/conftest.py

```diff
@@ -74,8 +74,8 @@ def domain_data_mock(hass: HomeAssistant) -> Iterable[Mock]:
     seal(upnp_device)
     domain_data.upnp_factory.async_create_device.return_value = upnp_device
 
-    with patch.dict(hass.data, {DLNA_DOMAIN: domain_data}):
```

Comment:
We should never patch `hass.data` in fixtures, because once the fixture is removed hass.data is reset and test cleanup is then very likely to fail